test/image/mocks/splom_multi-axis-type.json

```diff
@@ -16,11 +16,13 @@
         },
         {
           "label": "bool",
-          "values": [false, true, true, true, false, true, false, false, false, true]
+          "values": [false, true, true, true, false, true, false, false, false, true],
+          "axis": {"type": "category"}
```

Comment:
Yeah, that's way cleaner! 🎉

Good call making an `axis` container, I can see us wanting to allow other axis settings (gridlines, tick spacing, tick text...) to be set at the dimension level in the future.

I started thinking, since it's (generally) going to apply this to 2 axes, perhaps it should be called `axes`... but then that (elsewhere in the schema) implies a list of axes rather than axis properties. So I think it's good the way you have it.